src/pages/api/user/register.ts

```diff
@@ -0,0 +1,35 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import {
+  createUser,
+  getUser,
+  registrationSchema,
+} from "@server/services/userService";
+
+const registrationHandler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+) => {
+  if (req.method !== "POST") {
+    res.status(405).json({ message: "Method not allowed." });
+    return;
+  }
+
+  const result = registrationSchema.safeParse(req.body);
+  if (!result.success) {
```

Comment:
Can result be null? If so, we should do a null check

Reply:
No result will never be null